src/rustdoc_schema.graphql

```diff
@@ -499,12 +499,18 @@ type Method implements Item & FunctionLike {
 A specific attribute applied to an Item.
 """
 type Attribute {
-  """
-  The textual representation of the attribute.
+  as_string: String!
+  is_inner: Boolean!
+  content: AttributeValue!
```

Comment:
`content` is an edge but the others are properties. Please document these fields and keep consistency in the schema with respect to how the properties and edges are formatted (e.g. blank line between properties and edges, comments, etc.).

Reply:
I'm sorry, I should have looked over other types in the schema and seen how they are formatted, I forgot about that... Of course it will be changed